test/e2e-tests/README.md

```diff
@@ -1,16 +1,33 @@
 # AI Edge testing
 
-Place for testing and to verify the MLOps pipelines are working correctly. These tests use the `k8s.io/client-go` package to interact with the cluster. Using `oc login` to log into the cluster as normal should mean no setup is needed to run the tests.
+Place for testing and to verify the MLOps pipelines are working correctly. These tests use the `k8s.io/client-go` package to interact with the cluster. Using `oc login` to log into the cluster as normal should mean little setup is needed to run the tests.
 
 A local install of the Go compiler is needed to run the tests. Go version `1.21` is required to run the tests.
 
+## Setup
+- Log into the target cluster using `oc login`. This will update your default `kubeconfig` for the tests to use
+- Create a S3 bucket with the models in the root directory
+- Create a quay.io robot
+
+The following enviroment varaibles are required to run the test setup and the tests themselves. If any of these are not set then the tests will not run. Read [here](../../pipelines/README.md#ai-edge-end-to-end-pipeline) for more context on how to set these up.
+
+- `AWS_SECRET_ACCESS_KEY` - Secret from AWS
+- `AWS_ACCESS_KEY_ID` - Access key from AWS
+- `S3_REGION` - Region of the bucket used to store the model
+- `S3_ENDPOINT` - Endpint of the bucket
+- `IMAGE_REGISTRY_USERNAME` - quay.io username
+- `IMAGE_REGISTRY_PASSWORD` - quay.io password
+- `IMAGE_REGISTRY_CREDENTIALS` - quay.io robot secret file
+- `S3_BUCKET` - Name of S3 bucket that has the model
+- `TARGET_IMAGE_REPO` - Image repository that built model will be pushed to
```

Comment:
```suggestion
- `TARGET_IMAGE_REPO` - Image registry namespace that the built model will be pushed to a repository in (the repository name will be the same as the model name).
```

The pipeline parameter is confusingly named (and described). My suggestion here is a more accurate description. That pipeline parameter will be replaced as part of [RHOAIENG-2758](https://issues.redhat.com/browse/RHOAIENG-2758), so I wouldn't bother changing anything else around it in this PR, apart from this description here since this is new.

Reply:
I renamed this to `TARGET_IMAGE_NAMESPACE`